README.md

```diff
@@ -32,7 +32,7 @@ different categories based on their relevance to operators. Typhon uses this
 information to craft user interfaces.
 
 ## Installation
-Recommended installation:
+Recommended installation on Linux:
 ```
 conda install typhon -c pcds-tag -c conda-forge
```

Comment:
This wasn't changed in your PR, but this install line grabs other stuff from `pcds-tag` that the user will not want, such as our non-standard `pyqt` build. This is fine on-site, but annoying for people off-site. I recommend switching `conda-forge` to be first here.

I want to kill this build but I'm being dragged down by `pmgr`.

Reply:
Astute point. Thanks